From daae1ae35e13bc8107dc97d9219dfb8e172d5d2a Mon Sep 17 00:00:00 2001
From: Frederik Rietdijk <fridh@fridh.nl>
Date: Tue, 14 Mar 2017 15:00:33 +0100
Subject: [PATCH] namespace fix

configparser broke other namespace packages
https://github.com/NixOS/nixpkgs/issues/23855#issuecomment-286427428
This patch seems to solve that issue.
---
 setup.py                  | 1 -
 src/backports/__init__.py | 6 ------
 2 files changed, 7 deletions(-)

diff --git a/setup.py b/setup.py
index 3b07823..63ed25d 100644
--- a/setup.py
+++ b/setup.py
@@ -42,7 +42,6 @@ setup(
     py_modules=modules,
     package_dir={'': 'src'},
     packages=find_packages('src'),
-    namespace_packages=['backports'],
     include_package_data=True,
     zip_safe=False,
     install_requires=requirements,
diff --git a/src/backports/__init__.py b/src/backports/__init__.py
index f84d25c..febdb2f 100644
--- a/src/backports/__init__.py
+++ b/src/backports/__init__.py
@@ -3,9 +3,3 @@
 
 from pkgutil import extend_path
 __path__ = extend_path(__path__, __name__)
-
-try:
-    import pkg_resources
-    pkg_resources.declare_namespace(__name__)
-except ImportError:
-    pass
-- 
2.11.1

